src/WingetCreateCLI/Commands/CacheCommand.cs

```diff
@@ -82,6 +93,15 @@ public override Task<bool> Execute()
                     {
                         Logger.Debug(Path.GetFileName(file));
                     }
+
+                    Console.WriteLine();
+                    string[] directories = Directory.GetDirectories(PackageParser.InstallerDownloadPath);
+                    Logger.InfoLocalized(nameof(Resources.DirectoriesFound_Message), directories.Length, PackageParser.InstallerDownloadPath);
+                    Console.WriteLine();
```

Comment:
It would be best to create a new resource string just for deleting directories even if they are almost identical. That way there is no confusion and this doesn't look like a bug.